File: B-example-solutions/obj-c/01/Assignment-01-05/Assignment-01-05/ViewController.m
//
//  ViewController.m
//  Assignment-01-05
//
//  Created by Philip Dow on 8/28/15.
//  Copyright (c) 2015 Phil Dow. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    
    NSString *text = @"http://news.google.com";
    NSURL *URL = [NSURL URLWithString:text];
    
    if ( URL ) {
        NSURLRequest *request = [NSURLRequest requestWithURL:URL];
        [self.webView loadRequest:request];
    }
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: B-example-solutions/swift/01/Assignment-01-05/Assignment-01-05/ViewController.swift
//
//  ViewController.swift
//  Assignment-01-05
//
//  Created by Philip Dow on 8/28/15.
//  Copyright (c) 2015 Phil Dow. All rights reserved.
//

import UIKit

class ViewController: UIViewController, UITextFieldDelegate {

    @IBOutlet weak var webView : UIWebView!

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let text = "http://news.google.com"
        let URL = NSURL(string: text)
        
        if let URL = URL {
            let request = NSURLRequest(URL: URL)
            webView.loadRequest(request)
        }
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}